src/client/components/Header.tsx

```diff
@@ -2,64 +2,56 @@ import React from 'react';
 import { css } from '@emotion/react';
 import { brand, space } from '@guardian/src-foundations';
 import { from } from '@guardian/src-foundations/mq';
-import { Container } from '@/client/components/Container';
 import { Logo } from '@guardian/source-react-components-development-kitchen';
-
-const floatRight = css`
-  /** 
-    If you're here wanting to add more items to the Header then consider
-    removing this and using flex for positioning instead
-  */
-  float: right;
-`;
+import { gridRow, manualRow, SpanDefinition } from '@/client/styles/Grid';
 
 const bottomMarginStyles = css`
-  margin-bottom: ${space[1]}px;
-  ${from.tablet} {
-    margin-bottom: ${space[2]}px;
-  }
-  ${from.desktop} {
-    margin-bottom: ${space[4]}px;
-  }
+  margin-bottom: ${space[4]}px;
 `;
 
 const topMarginStyles = css`
-  margin-top: ${space[2]}px;
+  margin-top: ${space[5]}px;
+  ${from.mobileMedium} {
+    margin-top: 14px;
+  }
   ${from.tablet} {
     margin-top: ${space[2]}px;
   }
-  ${from.desktop} {
-    margin-top: ${space[3]}px;
-  }
-`;
-
-const rightMarginStyles = css`
-  margin-right: ${space[3]}px;
-  ${from.mobileLandscape} {
-    margin-right: ${space[5]}px;
-  }
-  ${from.wide} {
-    margin-right: ${space[24]}px;
-  }
 `;
 
 const backgroundColor = css`
   background-color: ${brand[400]};
 `;
 
+const headerGridRightToLeft = css`
+  direction: rtl;
```

Comment:
hadn't seen that before, neat!

Reply:
It's actually used for displaying languages that write from right to left, but we can repurpose it here, for the header element only as there is no text displayed.